htdocs/compta/bank/class/account.class.php

```diff
@@ -54,14 +55,6 @@ class Account extends CommonObject
     var $clos;
     var $rappro=1;    // If bank need to be conciliated
     var $url;
-    //! BBAN field for French Code banque
-    var $code_banque;
```

Comment:
For french people, four fields exists for RIB system. (code_banque | code_guichet | number | cle_rib)

This system is deprecated from august 2014. It's replaced by IBAN/BIC system in European Area.